src/patternfly/components/Check/styles.scss

```diff
@@ -1,8 +1,6 @@
 @import "../../patternfly-utilities";
 
 .pf-c-check {
-  --pf-c-check--MarginRight: var(--pf-global--spacer--md);
-  margin-right: var(--pf-c-check--MarginRight);
   cursor: pointer;
 
   &:disabled,
```

Comment:
This might be out of scope for your PR, but thought I'd throw this out there anyway. Right now, the only real indication I get that a radio or checkbox is disabled is when I hover over and get the `not-allowed` cursor. We could wrap checkbox and radio labels with `<span>` and extend the selector to include `.pf-c-check:disabled + *` to include the cursor change for the label as well. 

I'm interested in thoughts on this, and I also can create a separate issue for this if needed.

Reply:
hey @jgiardino I've updated this now, there was an extra "-" in the handlebars modifier. Let me know if this doesn't cover what you are thinking.

Reply:
yep, works now 👍